src/util/taproot.rs

```diff
@@ -131,70 +131,73 @@ impl TapLeafHash {
     }
 }
 
-/// Maximum depth of a Taproot Tree Script spend path
+/// Maximum depth of a taproot tree script spend path.
```

Comment:
nit, but not sure:
```suggestion
/// Maximum depth of a taproot tree script spending path.
```

Reply:
I believe 'key spend' and 'script spend' are the terms being used across the eccosystem, I could be wrong.